src/init.c

```diff
@@ -513,6 +513,13 @@ static void jl_uv_exitcleanup_walk(uv_handle_t *handle, void *arg)
 void jl_write_coverage_data(void);
 void jl_write_malloc_log(void);
 
+static struct uv_shutdown_queue_item *next_shutdown_queue_item(struct uv_shutdown_queue_item *item)
+{
+    struct uv_shutdown_queue_item *rv = item->next;
+    free(item);
+    return rv;
+}
```

Comment:
i realize this is strictly speaking correct, but is it really necessary? technically, everything in the GC is also being leaked at this point.